admin/javascript/src/sections/campaign-admin/tests/campaign-admin-test.js

```diff
@@ -0,0 +1,5 @@
+jest.dontMock('../controller');
+
+describe('CampaignAdminContainer', () => {
+
+});
```

Comment:
Yeah, the container isn't doing very much, all the meat is either in GridField, or yet to be built (preview, item list, etc). We could have more test coverage, but we're also still in the experimentation stage as you can see from the discussions on this pull request - so trying to strike a responsible balance between unit test coverage and fast refinement of the architecture.